chrome/content/zotero/fileInterface.js

```diff
@@ -76,6 +76,8 @@ Zotero_File_Exporter.prototype.save = function() {
 			translation.setCollection(this.collection);
 		} else if(this.items) {
 			translation.setItems(this.items);
+		} else {
+			translation.setLibraryID(this.libraryID);
```

Comment:
How would we distinguish between main library export and libraryID not being set?